pkg/globalconfig/global_config.go

```diff
@@ -63,6 +63,8 @@ type GlobalConfig struct {
 	UseTraefik                   bool                    `yaml:"use_traefik"`
 	WSL2NoWindowsHostsMgt        bool                    `yaml:"wsl2_no_windows_hosts_mgt"`
 	ProjectList                  map[string]*ProjectInfo `yaml:"project_info"`
+	RouterHTTPPort               string                  `yaml:"router_http_port"`
```

Comment:
This should go *above* the project_info member, otherwise they're hidden below that.